web/impact/impact/v1/helpers/matching_criterion_helper.py

```diff
@@ -5,6 +5,15 @@
 
 
 class MatchingCriterionHelper(CriterionHelper):
+    '''Helper for MatchingCriteria. MatchingCriteria specify a parameter on
```

Comment:
Obligatory [PEP 257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings) blank-line request.

Thanks for explaining matching criteria!